kotlin-react-dom/src/main/kotlin/react/dom/hydrate.kt

```diff
@@ -6,8 +6,10 @@ package react.dom
 import org.w3c.dom.Element
 import react.ReactElement
 
-// See https://reactjs.org/docs/react-dom.html
-
+/**
+ * [Deprecation details](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#deprecations)
+ */
+@Deprecated("Will be removed soon!")
```

Comment:
But this is not a truth. React team will not remove it soon.

Reply:
I agree, let's mark as deprecated, but not as planned for removal.

Reply:
Done